### PR TITLE
hardening/893_add_check_for_agent_file_name

### PR DESCRIPTION
**Pull request by pcoello25**
* Partially fix issue #893 
  * Update CHANGES_NEXT_RELASE is not neccesary.
* 100% Unit tests passed:
```
Results : Tests run: 67, Failures: 0, Errors: 0, Skipped: 0
```
* e2e (unofficial) tests passed:
```
[VALID AGENT FILE NAME: Example with GET method] 
{"success":"true","result" : {"cygnusagent.sinks.mysql-sink.data_model":"dm-by-attribute"}

[INVALID AGENT FILE NAME: ` agente_cygnus.conf`]
{"success":"false","error":"Agent file name must start with 'agent_'."}
```

### CODE REVIEW

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java [hunk 1]
```diff
@@ -626,6 +626,16 @@ protected void handleGetAdminConfigurationAgent(HttpServletRequest request, Http
         boolean allParameters = false;
         
         String param = request.getParameter("param");
+        String url = request.getRequestURI();
+     
+        String fileName = getFileName(url);
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: White line after this line (and remove the white line above :))

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java [hunk 2]
```diff
@@ -963,6 +973,16 @@ protected void handlePostAdminConfigurationAgent(HttpServletRequest request, Htt
         
         String param = request.getParameter("param");
         String newValue = request.getParameter("value");
+        String url = request.getRequestURI();
+     
+        String fileName = getFileName(url);
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Same comment than above.

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java [hunk 3]
```diff
@@ -1177,6 +1197,16 @@ private void handleDeleteAdminConfigurationAgent (HttpServletRequest request, Ht
         response.setContentType("json;charset=utf-8");
         
         String param = request.getParameter("param");
+        String url = request.getRequestURI();
+     
+        String fileName = getFileName(url);
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Same comment than above

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java [hunk 4]
```diff
@@ -1472,6 +1502,16 @@ protected void handlePutAdminConfigurationAgent(HttpServletRequest request, Http
         
         String param = request.getParameter("param");
         String newValue = request.getParameter("value");
+        String url = request.getRequestURI();
+     
+        String fileName = getFileName(url);
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Same comment than above

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java (hunk 1 above)
Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Fixed in 9755c0b

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java (hunk 2 above)
Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Fixed in 9755c0b

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java (hunk 3 above)
Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Fixed in 9755c0b

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java (hunk 4 above)
Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Fixed in 9755c0b

**Comment by frbattid** [comment]
LGTM

**Pull request merged**